HTML/packages.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tour Packages | Southern Taprobane Tours</title>
+    <link rel="stylesheet" href="../CSS/packages.css">
+</head>
+<body>
+    <header>
```

Comment:
Shoudn't we have the same navigation bar in these pages to make the website consistent? You can just copy paste nav bar code and it will appear here.